lib/matplotlib/axis.py

```diff
@@ -949,8 +955,62 @@ def set_tick_params(self, which='major', reset=False, **kwargs):
 
         self.stale = True
 
+    def get_tick_params(self, which='major'):
+        """
+        Get appearance parameters for ticks, ticklabels, and gridlines.
+
+        Parameters
+        ----------
+        which : {'major', 'minor'}, default: 'major'
+            The group of ticks for which the parameters are retrieved.
+
+        Returns
+        -------
+        dict
+            Properties for styling *new* elements added to this axis.
```

Comment:
OK, reading this I finally understand what you mean by "default style".  But I think that is incorrect terminology, and more confusing than helpful.  I think if someone has manually set a tick to a different styling, they will know they did it.  
```suggestion
            Properties for styling tick elements added to the axis.
```